pages/main.html

```diff
@@ -1,70 +1,54 @@
-  >
-    <div class="h1 font-semibold mb-8 text-shadow-lg">Local Logan</div>
-    <div class="text-3xl mb-4">Your one-stop shop for everthing local</div>
-    <div class="text-sm">Updated Weekly</div>
-    <div class="text-sm flex items-center">
-      This website has dark / light mode. It default to your systems preferences
-      but you can toggle here:&nbsp;&nbsp;
-      <div class="theme-toggle">
-        <label id="light-toggle" title="Use light theme" class="">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="20"
-            height="20"
-            viewBox="0 0 20 20"
-            fill="currentColor"
-          >
-            <path
-              fill-rule="evenodd"
-              d="M10 2a1 1 0 011 1v1a1 1 0 11-2 0V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95l.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707a1 1 0 11-1.414-1.414l.707-.707a1 1 0 011.414 0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707a1 1 0 00-1.414 1.414l.707.707zm1.414 8.486l-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z"
-              clip-rule="evenodd"
-            ></path>
-          </svg>
-          <input
-            type="radio"
-            name="theme-toggle"
-            value="light"
-            aria-label="Use light theme"
-            onclick="darkModeToggle()"
-          />
-        </label>
-        <label id="dark-toggle" title="Use dark theme" class="">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="20"
-            height="20"
-            viewBox="0 0 20 20"
-            fill="currentColor"
-          >
-            <path
-              d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
-            ></path></svg
-          ><input
-            type="radio"
-            name="theme-toggle"
-            value="dark"
-            aria-label="Use dark theme"
-            onclick="darkModeToggle()"
-          />
-        </label>
-      </div>
-    </div>
-  </div>
-  <iframe
-    src="https://calendar.google.com/calendar/embed?height=800&wkst=1&bgcolor=%23ffffff&ctz=America%2FDenver&src=MmI5ajk4YnZmZXZpOW1icnJjZ2FoYXE3amdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%23616161"
-    style="border: solid 1px #777"
-    class="w-screen h-screen my-4"
-    frameborder="0"
-    scrolling="no"
-  ></iframe>
-  <nav class="social-media-links flex justify-center space-x-4">
-    <img src="./assets/images/facebook.png" class="w-10 h-10" />
-    <!-- <img src="./images/messenger.svg" /> -->
-    <img src="./assets/images/instagram.png" class="w-10 h-10" />
-    <img src="./assets/images/twitter.png" class="w-10 h-10" />
-  </nav>
+	<div class="flex-1 shrink"></div>
+	<div class="basis-1/2 flex flex-col flex-grow items-center text-center align-middle">
```

Comment:
Ya' know, I really tried to get a pretty looking git diff but I think someone must have special tab/space rules that none of my editors have. Haha oops! 😅